web/apps/projets-citoyens/src/style/containers/citizen-project/_citizen-project-search.scss

```diff
@@ -169,11 +170,14 @@ a.citizen__project__item {
             background-color: white;
             border: 1px solid $gray--lighter;
             padding: 15px 20px;
-
             flex: 1;
-
             display: flex;
             flex-direction: column;
+
+            .district  {
+                color: #8d9aff ;
```

Comment:
Useless whitespace here =>  `color: #8d9aff;`

We don't have a color variable for this color ?